it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -331,6 +333,34 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
 
         }
       }
+      "return a redirect to continueUrl" when {
+        "the feature switch is enabled and business verification check is disable" in {
+          enable(EnableUnmatchedCtutrJourney)
+          await(insertJourneyConfig(
+            journeyId = testJourneyId,
+            authInternalId = testInternalId,
+            continueUrl = testContinueUrl,
+            optServiceName = None,
+            deskProServiceId = testDeskProServiceId,
+            signOutUrl = testSignOutUrl,
+            businessEntity = LimitedCompany,
+            businessVerificationCheck = false
+          ))
+          stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+          stubAudit()
+          stubRetrieveCompanyProfileFromBE(testJourneyId)(status = OK, body = Json.toJsObject(testCompanyProfile))
+          stubRetrieveCtutr(testJourneyId)(status = OK, body = testCtutr)
+          stubValidateIncorporatedEntityDetails(testCompanyNumber, testCtutr)(OK, Json.obj("matched" -> true))
+          stubStoreIdentifiersMatch(testJourneyId, identifiersMatch = true)(status = OK)
+
+          lazy val result = post(s"$baseUrl/$testJourneyId/check-your-answers-business")()
+
+          result.status mustBe SEE_OTHER
+          result.header(LOCATION) mustBe Some(routes.JourneyRedirectController.redirectToContinueUrl(testJourneyId).url)
+          verifyStoreIdentifiersMatch(testJourneyId, identifiersMatch = true)
+          verifyAudit()
+        }
+      }
```

Comment:
This should register, if it hands back to the calling service then no registration will have happened.